src/components/Footer/Filter.tsx

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames';
+import { FilterTypes } from '../../types/enums';
+import React from 'react';
+
+type Props = {
+  filterBy: FilterTypes;
+  setFilterBy: (filterBy: FilterTypes) => void;
```

Comment:
Let's create a handler function in case where we want to use a setter instead of using this setter. It will be a good practice and improve code readability

For example in this case you need to create a function in your app component that will be named like `handleChangeFilter`
and in Props you need to pass onChangeFilter function